distribution/std-lib/Base/src/Data/Ordering.enso

```diff
@@ -9,8 +13,15 @@ from Base import all
    `that`. So, if `this` is greater than `that`, you should return `Greater.`
 type Ordering
     ## An ordering where a compared value is less than another.
-    type Less
+    Less
     ## An ordering where a compared value is equal to another.
-    type Equal
+    Equal
     ## An ordering where a compared value is greater than another.
-    type Greater
+    Greater
+
+    ## Converts the ordering to the Java notion of ordering based on integers.
+    to_java : Integer
```

Comment:
Please change the name of this to something like `to_sign` or `to_integer`. It's not java specific at all, many languages use this form.